data-structure/hashTable.ts

```diff
@@ -0,0 +1,104 @@
+// Read about hash tables and its working mechanism and implement it functionality.
+// NOTE: get acquainted with possible collisions when working with hash tables, take this information into account when implementing.
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+type Item = string | number;
+
+type List = Array<Array<string | Item>>;
+
+type Store = Array<List>;
+
+class HashTable {
+  private readonly store: Store;
+
+  private readonly size: number;
+
+  constructor(size = 13) {
+    this.size = size;
+    this.store = [];
+  }
+
+  add(key: string, value: Item) {
+    const index: number = this.hash(key);
+
+    if (!this.store[index]) {
+      this.store[index] = [[key, value]];
+    } else {
+      const list: List = this.store[index];
+      const matchingIndex: number | void = HashTable.findMatchingIndex(
```

Comment:
why static? and if this method uses as util function and you don't want it to be available for instance add suitable access modifier